packages/flutter_tools/test/src/testbed.dart

```diff
@@ -700,6 +700,7 @@ class TestFeatureFlags implements FeatureFlags {
     this.isWebEnabled = false,
     this.isWindowsEnabled = false,
     this.isAndroidEmbeddingV2Enabled = false,
+    this.isIncrementalCompilerEnabled = false,
```

Comment:
Does it matter that this is disabled in the test above?

Reply:
It should be enabled, or it would not pass

Reply:
Ohh. You're right. I missed the `FeatureFlags` entry in the `overrides`. Will this test fail on non-master branches?

Reply:
No, since the TestFeatureFlags don't look in the environment